### DISCUSSION TITLE
What's the difference between displacement and vectors?

### DISCUSSION THREAD

**Discussion by pobo95** [ACEsuit/mace#373] [Q&A] [answered]
Thank you for your great work!

I'm studying your code these days.

So what is the difference between `displacement` and `vectors`?

It seems like 

$$vectors=r_j-r_i$$

but I have no idea about displacement, can you give me some help?

Thanks!

**Comment by ilyes319** [comment, marked as answer]
Displacement is displacing the cell. It used to compute  the stresses/virials.

**Reply by bernstei** [reply]
Those names are, if anything, the opposite of what I'd expect. I strongly suggest changing them. "vectors" sounds more like "cell vectors", and I know that atomic positions are also vectors, but that word isn't used explicitly for them as much. And "displacement" really suggests displacing atoms.

Can we add "position" or "pos" to "vectors", and maybe change "displacement" to "cell_deformation" or something? Is it the deformation gradient, in fact?

**Reply by pobo95** [reply]
I checked 
stresses/virials is

$$d(energy) \over d(displacement)$$

then `displacement` means changes of cells?
